kdc/kerberos5.c

```diff
@@ -1949,6 +1949,7 @@ _kdc_as_rep(astgs_request_t r)
     const char *msg;
     hdb_entry_ex *krbtgt = NULL;
     Key *krbtgt_key;
+    int client_reply_kvno;
```

Comment:
Not to be too fussy but, should be `unsigned int` to match definition in `hdb_asn1.h`.

Reply:
Or `krb5_kvno`.